package.json

```
@@ -52,9 +52,6 @@
     "testdouble-chai": "^0.2.2",
     "tmp": "0.0.28"
   },
-  "engines": {
-    "node": ">=0.10.40"
-  },
   "homepage": "https://github.com/spark/particle-commands/",
```

Let's take this opportunity to update the github URLs in package.json
```suggestion
  "homepage": "https://github.com/particle-iot/particle-commands/",
```